impeller/docs/glossary.md

```diff
@@ -0,0 +1,41 @@
+# Glossary
+
+### Device & Host
+
+In the context of graphics and Impeller, the device is the GPU and the host, the CPU.
+
+### Client Rendering API
+
+The API Impeller uses to talk to devices. Examples of these are OpenGL, Metal, Vulkan, Direct X.
+
+### Window System Integration (WSI)
+
+Impeller can render using one of the client rendering APIs into a render target. However, that render target needs to be presented in the platforms window system. A Window System Integration API does this and is usually extremely platform specific. For instance, OpenGL may be available on macOS and Linux, but the WSI API for macOS EAGL and usually (but not always) EGL for Linux.
+
+### Varying
+
+In the context of shaders, a varying is a value that is interpolated between two vertices (specified by the vertex shader) and provided to the fragment shader.
+
+### OpenGL
+
+[OpenGL and OpenGL ES (Embedded Systems)](https://www.opengl.org/) are [client rendering APIs](#client-rendering-api). These uses these on older versions of Android today.
```

Comment:
not quite sure what the last sentence of this paragraph is trying to say but it doesn't say it :-)

Reply:
Oops.